logrus.go

```diff
@@ -0,0 +1,85 @@
+package sched
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type LruLogger struct {
```

Comment:
Type should be private to avoid confusion.

Reply:
Done :)